bin/heroics-generate

```diff
@@ -26,15 +30,32 @@ option_parser = OptionParser.new do |opts|
 end
 
 option_parser.parse!
-if ARGV.length != 3
-  puts option_parser
-else
-  module_name, schema_filepath, url = ARGV
-  schema = Heroics::Schema.new(MultiJson.decode(open(schema_filepath).read))
-  cache = 'Moneta.new(:Memory)'
-  if options[:cache_path]
-    cache = "Moneta.new(:File, dir: \"#{options[:cache_path]}\")"
+
+case ARGV.length
+when 1
+  config_file = ARGV[0]
+  require config_file
+
+  puts Heroics.generate_client
+when 3
```

Comment:
I know it's obvious now, but it might be nice to throw in some comments or even an extracted function that makes it totally evident the difference between these code paths.